static_src/components/users_selector.jsx

```diff
@@ -11,6 +11,8 @@ import { Form, FormSelect } from './form';
 import PanelDocumentation from './panel_documentation.jsx';
 import userActions from '../actions/user_actions';
 import { validateString } from '../util/validators';
+import SpaceStore from '../stores/space_store.js';
```

Comment:
i appreciate the feedback. will do! just got the first version working, refactoring now.